src/image-viewer.js

```diff
@@ -490,23 +499,49 @@ export class ImageViewer {
   }
 
   /**
-   * Performs a one-step zoom action.
+ * Performs a one-step pinch zoom action.
+ * @param {number} centerClientX
+ * @param {number} centerClientY
+ * @param {number} deltaX
+ * @param {number} deltaY
+ *  @param {number} dir
+ * @private
+ */
```

Comment:
This is just a wrapper for naming conventions. We could also take it out and use `onZoomInc`. I'm not actually sure why it's called `onZoomInc`. Maybe `onZoomEvent` / `onZoomAction` would be more appropriate? Feedback welcome. =)

Reply:
Yeah `onZoomInc` is probably not the best name. I generally prefer to avoid abbreviations in variable and function names for this reason. I guess it was `onZoomIncrement`?
Maybe a better name could describe the action that the function accomplishes. `zoomToPoint`? `zoomInstant`? Naming is hard : P